File: C++/771. Jewels and Stones.cpp
class Solution {
public:
  int numJewelsInStones(string J, string S) {
    int res = 0;
    set<char> setJ(J.begin(), J.end());
    for(char s : S)
      if(setJ.count(s))
        res++;
    return res;
  }
};

File: Algorithms/Java/771. Jewels and Stones.java
class Solution {
  public int numJewelsInStones(String J, String S) {
    int res = 0;
    Set setJ = new HashSet();
    for(char j : J.toCharArray())
      setJ.add(j);
    for(char s : S.toCharArray())
      if(setJ.contains(s)) res++;
    return res;
  }
}
